### PR TITLE
Backpack/ minimal banner & dark mode

### PR DESCRIPTION
**Pull request by eadams17**
## WHAT
add minimal banner to Backpack and dark mode for banners

## WHY
we want this as a reusable component

## HOW
tweak `PostNavigationBanner` to accept `minimal` and 'dark-mode' styles (also tweak some of the CSS around button styling for existing banners)

### Screenshots
(If applicable. Also, please censor any sensitive data)
Screen Shot 2024-10-16 at 7 13 35 PM
Screen Shot 2024-10-17 at 3 39 24 PM

### Notion Card Links
(Please provide links to any relevant Notion card(s) relevant to this PR.)
https://www.notion.so/quill/Quill-Backpack-Addition-of-New-Banner-Minimal-Component-6a241fd27f2644caa12263b729c2e7d7

### What have you done to QA this feature?
(Provide enough detail that a reviewer could assess whether additional QA should be done. For larger projects, additionally use the Engineer Feature Testing Notion template. Review Guidelines if needed: https://www.notion.so/quill/Github-PR-QA-Guidelines-49e99fc965654ceeb8c6249bd9d181d7)
tested on staging that banner renders as expected (also verified that existing banners render as expected)

PR Checklist | Your Answer
------------ | -------------
Have you added and/or updated tests? | yes
Have you deployed to Staging? | yes
Self-Review: Have you done an initial self-review of the code below on Github? | yes

### CODE REVIEW

File: services/QuillLMS/client/app/bundles/Shared/styles/banner.scss
Comment (brendanshean) [inline services/QuillLMS/client/app/bundles/Shared/styles/banner.scss:1]: Nice refactoring here.

**Review by brendanshean — COMMENTED** [review]
Thank you for cleaning up this banner code.

File: services/QuillLMS/client/app/bundles/Shared/components/shared/__tests__/__snapshots__/postNavigationbanner.test.tsx.snap [hunk 1]
```diff
@@ -3,7 +3,7 @@
 exports[`PostNavigationBanner it should render 1`] = `
 <DocumentFragment>
   <div
-    class="banner-container green"
+    class="banner-container green green"
```
Comment (emilia-friedberg) [inline services/QuillLMS/client/app/bundles/Shared/components/shared/__tests__/__snapshots__/postNavigationbanner.test.tsx.snap:6]: NBD, but it seems a little strange to me that we wound up with an extra `green` here.

**Review by emilia-friedberg — APPROVED** [review]
These look great!

File: services/QuillLMS/client/app/bundles/Shared/components/shared/__tests__/__snapshots__/postNavigationbanner.test.tsx.snap (hunk 1 above)
Comment (eadams17) [reply services/QuillLMS/client/app/bundles/Shared/components/shared/__tests__/__snapshots__/postNavigationbanner.test.tsx.snap:6]: good catch! I forgot to update the props for this example

**Pull request merged**